app/services/entitlement_read_service.py

```
@@ -0,0 +1,557 @@
+            EntitlementEvent.acquisition_type == "PURCHASE",
+        )
+        .scalar() or 0
+    )
+    return purchase_count
+
+
+def get_active_rent_grant(
+    seat_id: int,
+    class_id: str,
+    product_id: int,
+    reference_time_utc: Optional[datetime] = None,
```

Ruff (0.16.0)

[warning] Unused function argument: `reference_time_utc`

(ARG001)